tests/unit/SpecialResolutionSummary.spec.ts

```diff
@@ -0,0 +1,75 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { SpecialResolutionSummary } from '@/components/SpecialResolution'
+import { getVuexStore } from '@/store/'
+import { createLocalVue, mount } from '@vue/test-utils'
+import { FeesIF } from '@/interfaces'
+
+Vue.use(Vuetify)
+
+const localVue = createLocalVue()
+const vuetify = new Vuetify({})
+
+describe('Special Resolution Review', () => {
+  let wrapper: any
+  let store: any = getVuexStore()
+
+  beforeEach(() => {
+    wrapper = mount(SpecialResolutionSummary, { vuetify, store, localVue })
+  })
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders multiple fees', async () => {
+    store.state.stateModel.currentFees = [{
+      'filingFees': 70.0,
+      'filingType': 'Special resolution',
+      'filingTypeCode': 'SPRLN',
+      'futureEffectiveFees': 0,
+      'priorityFees': 0,
+      'processingFees': 0,
+      'serviceFees': 0,
+      'tax': {
+        'gst': 0,
+        'pst': 0
+      },
+      'total': 70.0
+    },
+    {
+      'filingFees': 70.0,
+      'filingType': 'Special resolution',
+      'filingTypeCode': 'SPRLN',
+      'futureEffectiveFees': 0,
+      'priorityFees': 0,
+      'processingFees': 0,
+      'serviceFees': 0,
+      'tax': {
+        'gst': 0,
+        'pst': 0
+      },
+      'total': 70.0
+    }] as FeesIF[]
+    await Vue.nextTick()
```

Comment:
You shouldn't need this after assigning state data, though you might need it after the mount itself.

Reply:
Or, if you need to wait for stuff to happen after you set this, you could try awaiting the commit (instead of direct-assigning)?

Or, just add a comment why you need to await the next tick.

Reply:
Comments confirmed, thanks.